### DISCUSSION TITLE
Global administration pannel?

### DISCUSSION THREAD

**Discussion by Adesfire** [lukevella/rallly#152] [Ideas]
Hi there!
I was trying to find the way to access the global admin board, and now I am wondering if there is one to start with?
Maybe I missed something, but I didn't see any doc about this and no discussion is talking about it, so here I am :)
Anyway, I love this project which is really neat! Can't wait to know more about it.

**Comment by lukevella** [comment]
Hi, is this different than #153?

**Comment by Adesfire** [comment]
Yes it's different, here I think about a way to customize the landing page, to see how many polls have been made, to clean old polls, and other settings you might implement later (2fa, login and restrictions, themes,etc.)

**Comment by jacobsamlarose** [comment]
Not to hijack this request 😅 but one thing I'd like to see from a global dashboard is a list of all proposed dates in order to avoid double-booking between different meetings...

**Comment by eddydc** [comment]
I’m in favor of a landing page for admins only, secured by whatever technology. I want to prevent that polls wil be created or changed without being really admins. I tried to put basic authentication on /admin and /new but that als blocked things on the poll page itself

**Comment by Adesfire** [comment]
Agreed, we could keep it simple with a basic authentication system.

**Reply by eddydc** [reply]
Tried it, but, as said, it gave issues with the regular poll page. It asks for authentication when setting authentication on /new and /admin

**Reply by Adesfire** [reply]
My bad, by basic I meant a regular password/username authentication stored in the current DB. In opposition with modern authentication, groups, users, and all the stuff
